openadapt/entrypoint.py

```diff
@@ -13,21 +13,19 @@
 
 def run_openadapt() -> None:
     """Run OpenAdapt."""
-    try:
-        from openadapt.alembic.context_loader import load_alembic_context
-        from openadapt.app import tray
-        from openadapt.config import print_config
+    with redirect_stdout_stderr():
+        try:
+            from openadapt.alembic.context_loader import load_alembic_context
+            from openadapt.app import tray
+            from openadapt.config import print_config
 
-        print_config()
-        load_alembic_context()
-        tray._run()
-    except Exception as exc:
-        data_dir = get_root_dir_path()
-        # TODO: log all exceptions to a file
-        with open(data_dir / "error.log", "a") as f:
-            f.write(f"{datetime.now()}: {exc}\n")
```

Comment:
Why was this removed? Why not log to the same file? e.g.

```
with redirect_stdout_stderr() as redirect:
    try:
        ...
    except:
        with open(redirect.log_file, "a") as f:
            ...
```

Reply:
Since we are overriding stdout, print will automatically write to that file

Reply:
Why `print` and not `logger.info`?